chart/inlets-operator/README.md

```diff
@@ -17,19 +17,33 @@ If you would like to use Helm instead, read on.
   helm repo update
   ```
 
+* Create a namespace for the inlets-operator
+
+  ```sh
+  kubectl create namespace inlets
+  ```
+
+* Create a secret for the inlets license key:
+
+  ```sh
+  kubectl create secret generic -n inlets \
+    inlets-license --from-file license=$HOME/.inlets/LICENSE
+  ```
+
 * Create your access key secret, then apply one of the following configurations.
 
 * Create a secret with an API key generated by DigitalOcean
 
     ```sh
-    kubectl create secret generic inlets-access-key \
-    --from-literal inlets-access-key="$(cat ~/Downloads/do-access-token)"
+    kubectl create secret generic -n inlets \
+    inlets-access-key \
+    --from-file inlets-access-key=$HOME/Downloads/do-access-token
     ```
 
 * Install the CRD:
```

Comment:
We could also consider letting helm install the CRD instead of this manual step?

Reply:
What about managing changes in the CRD?

Reply:
I think upgrading the CRD would still have to be done manually as Helm does not support that.